strategy/overall-aims.md

```diff
@@ -0,0 +1,22 @@
+# Human Cell Atlas Data Coordination Platform - Our Strategy
+
+## Mission
+To provide a platform for biological scientists to share, organise, analyse, evaluate and access single-cell data, helping them create comprehensive reference maps of all human cells.
+
+## Overall Aims
+
+### Aim 1
+**Create a data resource that maximizes the value and use of Human Cell Atlas data across the scientific community.**
+All data considered to be suitable towards building the Human Cell Atlas will be collected and organized in a single repository in accordance with FAIR data principles. We will collect experimental protocols, raw data, and processed data with thorough documentation of provenance, represented in structured metadata and additional supplemental files. We will capture all relevant data by any possible means, including extraction of relevant raw data from other archives. We will comply with ethical and legal requirements for data access, as well as being sensitive to the privacy concerns of researchers, whilst seeking to ensure data is openly available for the entire community. [Read more...](./specific-aims.md#aim-1)
+
+### Aim 2
+**Provide researchers access to data that will enable the Human Cell Atlas to answer fundamental questions in all aspects of biology.** Focusing primarily on “researchers with a keyboard” and “researchers with a pipette”, we will provide programmatic interfaces (APIs) and graphical user interfaces that allow these scientists to discover, integrate and access datasets relevant to their research efforts. Informed by the development of the Human Cell Atlas, tissue-specific atlases and their requirements, we will focus on presenting data from the DCP with a core set of features sufficient to characterize cell types using currently available methods. [Read more...](./specific-aims.md#aim-2)
```

Comment:
"data from the DCP" --> "HCA data"

Reply:
Hey Jason - I'm definitely on vacation here! One thought on this: are you thinking these are equivalent or are you aiming to narrow the scope? I'd see "data from the DCP" to be a superset of "HCA data" (i.e. not all the contents of the DCP will be used to create the reference atlas). If we're narrowing down (so we only focus on presenting HCA data) I think this would be a good thing, but we probably need to say how we _know_ which data is eligible

Reply:
I see these as equivalent and merely semantic changes. I don't think "DCP data" is a thing...unless someone here has a secret life (how's your "vacation" going, Tony?). I would be OK specifying "HCA data deposited with the DCP" or "...hosted by the DCP". So I did consider the 3 cases below in bigger need of re-phrasing.

Reply:
I guess I was thinking about it more as a lifecycle thing - single cell data is submitted to the DCP and from there, some of it may "end up" in the HCA rather than being "born" as HCA data. I don't have any particularly strong feelings about the phrasing though, I'd be happy with the change

Reply:
The definition in my head is that "HCA data" is anything that is accessible at https://data.humancellatlas.org/ because I think that's how users will define it.

Reply:
I think there are actually three categories here, one of which is simple and two of which are more complex and share a name

DCP data is easy, data hosted on the DCP 
HCA data though can and will be used to mean two things, especially right now, data collected to support the aims of the HCA and then the specific data which is used to build the atlas

Not entirely sure it is worth trying to pry these two concepts apart in this strategy as we need to support both classes of HCA data

Reply:
> The definition in my head is that "HCA data" is anything that is accessible at https://data.humancellatlas.org/

Lol, when you put it like that, it's pretty convincing.

Reply:
> The definition in my head is that "HCA data" is anything that is accessible at https://data.humancellatlas.org/ because I think that's how users will define it.

+1